BookingService/BookingService.Infrastructure/Authentication/JwtTokenService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using BookingService.Domain;
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+
+namespace BookingService.Infrastructure.Authentication
+{
+    internal class JwtTokenService : ITokenService
+    {
+        private readonly IConfiguration _configuration;

```

Comment:
Use `IOptions` pattern and inject only values that u need here. Otherwise this class can use any configuration, even config which it is not need. This is Separation of Concerns practice